intellij-ext/modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordConfigDropDown.kt

```diff
@@ -0,0 +1,65 @@
+package com.metalbear.mirrord
+
+import com.intellij.openapi.actionSystem.*
+import com.intellij.openapi.actionSystem.ex.ComboBoxAction
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.newvfs.BulkFileListener
+import com.intellij.openapi.vfs.newvfs.events.VFileEvent
+import javax.swing.JComponent
+
+class MirrordConfigDropDown : ComboBoxAction() {
+    companion object {
+        private var project: Project? = null
+        private val configPaths by lazy { arrayListOf<String>() }
+        private var shouldUpdatePaths = true
+        var selectedConfig = "Select Configuration"
+
+        fun updatePaths() {
+            project?.takeIf { it.isInitialized }?.let { prj ->
+                val paths = MirrordConfigAPI.searchConfigPaths(prj)
+                configPaths.clear()
+                configPaths.addAll(paths)
+                if (configPaths.isNotEmpty()) {
+                    if (!paths.contains(project?.basePath + "/.mirrord/mirrord.json")) {
+                        configPaths.add(0, "Default Configuration")
+                    }
+                    selectedConfig = "Select Configuration"
```

Comment:
I don't think we want to have `Select configuration` option rather just have Default Configuration as the default selected, or the first one that is found if not the default.